### PR TITLE
fix AuthLayout height for correct position of reistration form

### PR DESCRIPTION
**Pull request by IvanRuskevych**

### CODE REVIEW

File: src/components/RegistrationForm/RegistrationForm.styled.js [hunk 1]
```diff
@@ -29,12 +29,14 @@ export const FormWrapper = styled(StyledFlexDiv)`
   background-color: var(--color-bg-form);
 
   @media ${device.laptop} {
+    margin: 12px 0;
```
Comment (Aleksey2892) [inline src/components/RegistrationForm/RegistrationForm.styled.js]: You cannot add margin to inside elements. Only padding. You can add padding to MainLayout to fix this issue

File: src/components/RegistrationForm/RegistrationForm.styled.js [hunk 2]
```diff
@@ -29,12 +29,14 @@ export const FormWrapper = styled(StyledFlexDiv)`
...
     padding: 0 25px;
     width: 658px;
     border-radius: 10px;
   }
 
   @media ${device.desktop} {
+    margin: 27px 0;
```
Comment (Aleksey2892) [inline src/components/RegistrationForm/RegistrationForm.styled.js]: Look the same rule that I wrote above

**Review by Aleksey2892 — CHANGES_REQUESTED** [review]
There are mistakes there.

File: src/components/RegistrationForm/RegistrationForm.styled.js (hunk 1 above)
Comment (IvanRuskevych) [reply src/components/RegistrationForm/RegistrationForm.styled.js]: Changed only the two margins (margin: 12px 0; margin: 27px 0;). I almost finish the new markup of registration form and this time used margin & paddings correctly .

File: src/components/RegistrationForm/RegistrationForm.styled.js (hunk 2 above)
Comment (IvanRuskevych) [reply src/components/RegistrationForm/RegistrationForm.styled.js]: Done

**Review by Aleksey2892 — CHANGES_REQUESTED** [review]
Well, that's OK now. But have you changed the eyes' color for icons? Waiting for it also

**Comment by Aleksey2892** [comment]
We can finally merge it :p

**Review by Aleksey2892 — APPROVED** [review]
We can finally merge it :p

**Pull request merged**